pravega/src/test/java/docs/javadsl/PravegaGraphTestCase.java

```diff
@@ -96,7 +96,7 @@ public void infiniteSourceTest() throws ExecutionException, InterruptedException
             .run(materializer);
     // #reading
 
-    Thread.sleep(3000);
+    Thread.sleep(2000);
```

Comment:
We can block on a `Promise` here and wait for some state before continuing, instead of a sleep.